website_and_docs/content/project/governance/_index.html

```diff
@@ -428,12 +428,19 @@ <h3 id="project-leadership-committee-plc">Project Leadership Committee (PLC)</h3
       </p>
 
       <p class="pb-2">
-        PLC seats are not time-limited. Only one person on the PLC may be affiliated with
+        Standard PLC seats are not time-limited. Only one person on the PLC may be affiliated with
         any given employer at a time. The PLC member count should be always
         odd so that no votes are ever tied, an ideal minimum number is 5. PLC members
         should also be active members of the community.
       </p>
 
+      <p class="pb-2">
```

Comment:
This language should be appended with information on how the voting in case of tie works. In many non-profits bylaws, the chair vote is considered to have winning emphasis on case of a tie, and if the TLC vote represents multiple people, it might make sense to either allow that or actively remove that.

Reply:
Addressed this in the recent commit.